File: 07-Jun-C-C++/Classwork/C Lecture Flow/18-6-22/ladderif-else.c
#include <stdio.h>

void main()
{
	char name[10];
	int rollno,s1,s2,s3,total;
	float per;
	char symbol='%';
	printf("\n- <-------Accept Student Detail----------->-\n");
	printf("\nEnter Roll NO. : ");
	scanf("%d",&rollno);
	printf("Enter Name : ");
	scanf("%s",&name);
	printf("Enter Physics Marks : ");
	scanf("%d",&s1);
	printf("Enter Chemistry Marks : ");
	scanf("%d",&s2);
	printf("Enter Maths Marks : ");
	scanf("%d",&s3);
	total = s1+s2+s3;
	per = (float)total/3; 
	printf("\nTotal PCM Marks : %d",total);
	printf("\nPercentage is %.2f%c",per,symbol);
	printf("\nGrade : ");
	if(per > 75)
	{
		printf("Distinction");
	}
	 if(per > 60)
	{
		printf("First Class");
	}
	 if(per > 50)
	{
		printf("Second Class");
	}
	if(per > 40)
	{
		printf("Pass Class");
	}
	else
	{
		printf("Fail");
	}
	
	printf("\n\n<--------Thank you--------->");
		
}

File: Java19July9-10.30/Java-Classwork/src/com/foundation/LadderIfElse.java
package com.foundation;

import java.util.Scanner;

public class LadderIfElse {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter Roll.No");
		int rollno = sc.nextInt();
		System.out.println("Enter Name");
		String name = sc.next();
		System.out.println("Enter Physics Marks");
		int p = sc.nextInt();
		System.out.println("Enter Chemistry Marks");
		int c = sc.nextInt();
		System.out.println("Enter Maths Marks");
		int m = sc.nextInt();
		int tot = p+c+m;
		float per = (float)tot/3;
		System.out.println("Roll No : "+rollno);
		System.out.println("Name : "+ name);
		System.out.println("Total is "+ tot);
		System.out.println("Percentage is "+per);
		
		if(per>75)
		{
			System.out.println("Distinction");
		}
		else if(per >60)
		{
			System.out.println("First Class");
		}
		else if(per > 50)
		{
			System.out.println("Second Class");
		}
		else if(per > 40)
		{
			System.out.println("Pass Class");
		}
		else
		{
			System.out.println("Fail");
		}
		
	}
}











